### PR TITLE
Reorder registry blog sections

### PR DESCRIPTION
**Pull request by thockin**
xref #39895

### CODE REVIEW

File: content/en/blog/_posts/2023-03-10-image-registry-change.md [hunk 1]
```diff
@@ -33,83 +33,24 @@ registry](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-g
...
-
-## How can I check registry.k8s.io is accessible from my cluster?
+## How can I check if I am impacted?
 
 To test connectivity to registry.k8s.io and being able to pull images from there, here is a sample
 command that can be executed in the namespace of your choosing:
 
 ```
-kubectl run hello-world --tty --rm -i --image=registry.k8s.io/busybox:latest sh
+kubectl run hello-world -ti --rm --image=registry.k8s.io/busybox:latest --restart=Never -- date
```
Comment (sftim) [inline content/en/blog/_posts/2023-03-10-image-registry-change.md:42]: nit: this change doesn't match the PR title

**Review by sftim — COMMENTED** [review]
/approve

Please cancel this approval if (and only if) making significant changes to the PR from how it looks right now.

File: content/en/blog/_posts/2023-03-10-image-registry-change.md (hunk 1 above)
Comment (BenTheElder) [reply content/en/blog/_posts/2023-03-10-image-registry-change.md:42]: +1 to this change however, seems like a worthwhile improvement.

**Review by BenTheElder — APPROVED** [review]

**Comment by dims** [comment]
thanks @thockin

File: content/en/blog/_posts/2023-03-10-image-registry-change.md (hunk 1 above)
Comment (thockin) [reply content/en/blog/_posts/2023-03-10-image-registry-change.md:42]: title updated :)

**Pull request merged**